src/lang/nodejs/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "blotjson",
+  "version": "0.1.0",
+  "description": "A module to simplify visualisation of JSON.",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest --forceExit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/arsalanc-v2/blotjson.git"
+  },
+  "author": "Saad Haider",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/arsalanc-v2/blotjson/issues"
+  },
+  "homepage": "https://github.com/arsalanc-v2/blotjson#readme",
+  "dependencies": {
+    "open": "^7.0.4",
+    "websocket": "^1.0.31",
+    "ws": "^7.3.0"
```

Comment:
Since _ws_ is only used in the tests, let's add it as a dev dependency. Also, why not use `.client `of _websocket_ instead of using _ws_?

Reply:
Oh yea I was having a lot of issues when I was writing the client code (until i found out about the beforeAll method which i had to use), during which I tried different libraries and forgot to refactor this part. Will change!

Reply:
Actually, after trying to refactor it, I have found ws to be much easier to use than websocket for the client side and I don't see any point in putting in the effort to debug everything to refactor it so I'll stick with it. I'll just add it as a dev dependency

Reply:
Alright I'll open an issue for this which can be addressed in the future.

Reply:
yeap ok. Anyway, this is just for the testing. On the actual frontend this isn't an issue since browser's have their own support for WebSocket, which basically works the same as the ws package